src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageTest {
+
+    @BeforeEach
+    void setUp() {
+        Map<String, Integer> fruits = Storage.getAllFruits();
+        fruits.keySet().forEach(fruit ->
+                Storage.modifyFruitStorage(fruit, -Storage.getFruitQuantity(fruit)));
+    }
+
+    @Test
+    void modifyFruitStorage_addPositiveQuantity_ok() {
+        Storage.modifyFruitStorage("apple", 50);
+        int actual = Storage.getFruitQuantity("apple");
+        assertEquals(50, actual, "Количество яблок должно быть 50");
+    }
+
+    @Test
+    void modifyFruitStorage_addZeroQuantity_ok() {
+        Storage.modifyFruitStorage("apple", 0);
+        int actual = Storage.getFruitQuantity("apple");
+        assertEquals(0, actual, "Количество яблок должно быть 0");
+    }
+
+    @Test
+    void modifyFruitStorage_addNegativeQuantity_ok() {
+        Storage.modifyFruitStorage("apple", 50); // Добавляем 50
+        Storage.modifyFruitStorage("apple", -20); // Уменьшаем на 20
```

Comment:
these comments aren’t informative. Let’s remove it. But if you want to leave it use the English